### DISCUSSION TITLE
Github e vscode

### DISCUSSION THREAD

**Discussion by emyllisoares** [community/community#120159] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Quando entro no meu VS code não consigo mais logar com a conta do GitHub, somente aparece conectar com a Mircrosoft, como resolver?

**Comment by queenofcorgis** [comment]
Hi @emyllisoares , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.

**Comment by queenofcorgis** [comment]
Please note for future posts:. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
